src/SIL.LCModel/DomainImpl/CmObject.cs

```diff
@@ -2894,7 +2894,7 @@ internal static GenDate GetGenDateFromInt(int nVal)
 		/// <param name="flid">Custom field id.</param>
 		/// <param name="newValue">New value. (May be null.)</param>
 		/// <param name="useAccessor">'true' to send to UOW. 'false' to not send to the UOW.</param>
-		private void SetCustomPropertyForSDA(int flid, object newValue, bool useAccessor)
+		public void SetCustomPropertyForSDA(int flid, object newValue, bool useAccessor)
```

Comment:
I didn't realize that you needed to make this public. There is probably a better way of getting this change made through a method that is already internal or protected.